### PR TITLE
Feat/adding mobula env

### PR DESCRIPTION
**Pull request by xyeuu**
- Adding mobula env

### CODE REVIEW

File: for-developers/information-and-settings/env-variables.md [hunk 1]
```diff
@@ -361,28 +361,32 @@ Please note that these zkSync-related variables are only supported together with
...
-| `EXCHANGE_RATES_COINGECKO_BASE_URL`                  |          | If set, overrides the Coingecko base URL. Implemented in [#9679](https://github.com/blockscout/blockscout/pull/9679).                                                                                                                                                  | true             | v6.4.0+ | API, Indexer  |
-| `EXCHANGE_RATES_COINGECKO_BASE_PRO_URL`              |          | If set, overrides the Coingecko Pro base URL. Implemented in [#9679](https://github.com/blockscout/blockscout/pull/9679).                                                                                                                                              | true             | v6.4.0+ | API, Indexer  |
+| Variable                                             | Required | Description                                                                                                                                                                                                                                                            | Default          | Version |
```
Comment (vbaranov) [inline for-developers/information-and-settings/env-variables.md]: The upstream branch has a new column `|Application |`. Your changes remove that column. Could you please pull updates from the master branch?

**Review by vbaranov — CHANGES_REQUESTED** [review]

File: for-developers/information-and-settings/env-variables.md [hunk 2]
```diff
@@ -361,28 +361,32 @@ Please note that these zkSync-related variables are only supported together with
...
+| `EXCHANGE_RATES_TVL_SOURCE`                          |          | This environment variable is used to set source for TVL fetching. Available value is `defillama`.                                                                                                                                                                      | (empty)          | v5.3.0+ |
+| `EXCHANGE_RATES_PRICE_SOURCE`                        |          | This environment variable is used to set source for price fetching. Available values are also `crypto_compare`, `coin_gecko`,`coin_market_cap` and `mobula`.                                                                                                                    | `crypto_compare` | v5.2.3+ |
+| `EXCHANGE_RATES_MOBULA_COIN_ID`                      |          | Explicitly set Mobula coin ID.                                                                                                                                                                                                                                      | (empty)          | v4.1.4+ |
```
Comment (sl1depengwyn) [inline for-developers/information-and-settings/env-variables.md]: Version for newly created envs should be `master`

**Review by sl1depengwyn — CHANGES_REQUESTED** [review]

File: for-developers/information-and-settings/env-variables.md (hunk 2 above)
Comment (xyeuu) [reply for-developers/information-and-settings/env-variables.md]: What is the master version ? How do I find it ? Thanks

Comment (sl1depengwyn) [reply for-developers/information-and-settings/env-variables.md]: You need to put `master` in the version field

**Review by vbaranov — APPROVED** [review]

**Pull request merged**